From d59321635f3eef6fd983dad345acb9d44d02949a Mon Sep 17 00:00:00 2001
From: neo <neo.dowithless@gmail.com>
Date: Fri, 13 Jun 2025 22:27:28 +0800
Subject: [PATCH] doc: Add links to translated README versions

Added language selection links to the README for easier access to translated versions:
German, Spanish, French, Portuguese, and Russian.
---
 README.md | 7 ++++++-
 1 file changed, 6 insertions(+), 1 deletion(-)

diff --git a/README.md b/README.md
index cd994bdc6c4..57fe673ef4f 100644
--- a/README.md
+++ b/README.md
@@ -11,7 +11,12 @@
 [![GitHub star chart](https://img.shields.io/github/stars/FlowiseAI/Flowise?style=social)](https://star-history.com/#FlowiseAI/Flowise)
 [![GitHub fork](https://img.shields.io/github/forks/FlowiseAI/Flowise?style=social)](https://github.com/FlowiseAI/Flowise/fork)
 
-English | [繁體中文](./i18n/README-TW.md) | [简体中文](./i18n/README-ZH.md) | [日本語](./i18n/README-JA.md) | [한국어](./i18n/README-KR.md)
+English | [繁體中文](./i18n/README-TW.md) | [简体中文](./i18n/README-ZH.md) | [日本語](./i18n/README-JA.md) | [한국어](./i18n/README-KR.md) | 
+[Deutsch](https://www.readme-i18n.com/FlowiseAI/Flowise?lang=de) | 
+[Español](https://www.readme-i18n.com/FlowiseAI/Flowise?lang=es) | 
+[français](https://www.readme-i18n.com/FlowiseAI/Flowise?lang=fr) | 
+[Português](https://www.readme-i18n.com/FlowiseAI/Flowise?lang=pt) | 
+[Русский](https://www.readme-i18n.com/FlowiseAI/Flowise?lang=ru) | 
 
 <h3>Build AI Agents, Visually</h3>
 <a href="https://github.com/FlowiseAI/Flowise">
